src/Aspire.Hosting.Docker/DockerComposeAspireDashboardResourceBuilderExtensions.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Docker;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for creating Aspire Dashboard resources in the application model.
+/// </summary>
+public static class DockerComposeAspireDashboardResourceBuilderExtensions
+{
+    /// <summary>
+    /// Creates a new Aspire Dashboard resource builder with the specified name.
+    /// </summary>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/> instance.</param>
+    /// <param name="name">The name of the Aspire Dashboard resource.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{AspireDashboardResource}"/>.</returns>
+    /// <remarks>
+    /// This method initializes a new Aspire Dashboard resource that can be used to visualize telemetry data
+    /// in the Aspire Hosting environment. The resource is not automatically added to the application model;
+    /// instead, it returns a builder that can be further configured and added as needed.
+    /// </remarks>
+    /// <exception cref="ArgumentNullException">Thrown when <paramref name="builder"/> is <c>null</c>.</exception>
+    /// <exception cref="ArgumentException">Thrown when <paramref name="name"/> is <c>null</c> or empty.</exception>
+    internal static IResourceBuilder<DockerComposeAspireDashboardResource> CreateDashboard(
+        this IDistributedApplicationBuilder builder,
+        string name)
+    {
+        ArgumentNullException.ThrowIfNull(builder);
+        ArgumentException.ThrowIfNullOrEmpty(name);
+
+        var resource = new DockerComposeAspireDashboardResource(name);
+
+        // Initialize the dashboard resource
+        return builder.CreateResourceBuilder(resource)
+                      .WithImage("mcr.microsoft.com/dotnet/nightly/aspire-dashboard")
```

Comment:
We aren't setting an image tag. Will the latest aspire dashboard always be compatible with older AppHost versions? For example, if the dev is on 9.3 and the latest aspire dashboard is `10`?

Reply:
This is forcing function for us to get our act together with the standalone dashboard image.

> We aren't setting an image tag. Will the latest aspire dashboard always be compatible with older AppHost versions? For example, if the dev is on 9.3 and the latest aspire dashboard is 10?

There's no app host here, this is just otel.